includes/admin/settings.php

```diff
@@ -87,6 +91,45 @@ public function column_payment_method_value( $value, $affiliate ) {
 		return $value;
 	}
 
+	/**
+	 * Add a "Payment Method" column to the referrals screen.
+	 * 
+	 * @since 2.3.3
+	 *
+	 * @param array  $prepared_columns Prepared columns.
+	 * @param array  $columns  The columns for this list table.
+	 * @param object $instance List table instance.
+	 * 
+	 * @return array $prepared_columns Prepared columns.
+	 */
+	public function referrals_column_store_credit( $prepared_columns, $columns, $instance ) {
+
+		$offset = 8;
+
+		$prepared_columns = array_slice( $prepared_columns, 0, $offset, true ) +
+		                    array( 'payout_method' => __( 'Payout Method', 'affiliatewp-store-credit' ) ) +
+		                    array_slice( $prepared_columns, $offset, null, true );
+
+		return $prepared_columns;
+	}
+
+	/**
+	 * Show the payment method for each referral.
+	 *
+	 * @since 2.3.3
+	 *
+	 * @param string $value     The column data.
+	 * @param object $affiliate The current affiliate object.
+	 *
+	 * @return string $value   The affiliate's payment method.
```

Comment:
```suggestion
	 * @return string The affiliate's payment method.
```
Remove the empty line between `@param` and `@return` notations, no variable for `@return` notations.